flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -1366,7 +1410,7 @@ public DynamicTableSource copy() {
      * scan source without lookup ability, e.g. testing temporal join changelog source.
      */
     private static class TestValuesScanLookupTableSource extends TestValuesScanTableSource
-            implements LookupTableSource {
+            implements LookupTableSource, SupportsDynamicFiltering {
```

Comment:
Only scan table source need to implements SupportsDynamicFiltering

Reply:
For `TestValuesTableSource`, they support two types of scan source named `TestValuesScanLookupTableSource` and `TestValuesScanTableSource`， they all support scan read. So I think this two source need to implements `SupportsDynamicFiltering`.